_drafts/autolab-and-docker.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Autolab and Docker"
+author: Mihir Pandya
+
+tags: [autograding, docker, tango]
+---
+
+![]({{site-url}}/assets/docker.svg)
+
+As highlighted in our ['Hello World' post][hello-world], we at Autolab want to make Autograding for all a reality. In pursuit of this vision, we started by making Autolab robust enough to be usable by a variety of classes. Following that, we [open-sourced][gh-autolab] this project after it had been tried and tested at Carnegie Mellon University. 
```

Comment:
- I would use the same case on both "Autograding" and "all":
  - Autograding for All, or
  - autograding for all
- robust enough __for__ a variety of classes
  - reduce wordiness